poros/game_results.rb

```diff
@@ -6,6 +6,7 @@ def service
   end
 
   def game_params_by_name(game_name)
+    return "Invalid Game Name" if service.get_games_by_name(game_name).empty?
```

Comment:
if game name is not a perfect match, we can now `fail gracefully`.  The goal all along